71-example-eks-clusterapi-base-cluster (#205)
arlonproj/arlon: README.md

If you make subsequent changes to the base cluster, workload clusters originally created from it
will automatically acquire the changes.

> Below is an example of an manifest file with which we can create a *base cluster*. This manifest file helps in 

Reviewer: ```suggestion
Here is an example of a manifest file that we can use to create a *base cluster*. This manifest file helps in 
```

> deploying an EKS cluster with machine deployment component.

Reviewer: ```suggestion
deploying an EKS cluster with 'machine deployment' component from the cluster API (CAPI).
```
